scripts/publish/combine_minify_js.R

```diff
@@ -8,6 +8,20 @@ publish.combine_minify_js <- function(viz) {
   
   # combine
   file_connection <- file(viz[["location"]])
+  
+  # first add vizlab resources
+  for(lib in args[["lib_js"]]) {
+    lib_viz <- as.viz(lib)
+    publish(lib)
+    # vizlab location puts these files in js/d3-modules/
+    # so need to change just to js/
+    lib_fp <- paste0(exportLocation(), "js/", basename(lib_viz[["location"]]))
```

Comment:
no need to change this, but is it better to use `paste0()` over `file.path()` for these situations?

Reply:
I couldn't use file path because `exportLocation()` returns a directory with the `/`

Reply:
I guess I could strip that off and then use file.path